screens/ProfileScreen/Navigation/ProfileScreensNavigation.tsx

```diff
@@ -1,23 +1,25 @@
-import React from "react"
-import { StackScreenProps, createStackNavigator } from "@react-navigation/stack"
-import ProfileScreenView, { ProfileScreenViewProps } from "../ProfileView"
 import { ChevronBackButton, StackNavigatorType } from "@components/Navigation"
-import { SettingsScreen } from "@screens/SettingsScreen/SettingsScreen"
-import { ActivitiesStackParamList } from "@stacks/ActivitiesStack"
-import { useAtomValue } from "jotai"
-import { userAtom } from "../state"
-import { UserMocks } from "@lib/users/User"
-import { HeaderLeftProfile, HeaderRightProfile } from "./ProfileHeaders"
 import { Headline } from "@components/Text"
+import { TouchableIonicon } from "@components/common/Icons"
 import { AppStyles } from "@lib/AppColorStyle"
-import { EditProfileDismissButton } from "./EditProfileDismissButton"
-import EditProfileView from "../EditProfileScreen/EditProfileView"
 import { EventMocks } from "@lib/events"
+import { UserMocks } from "@lib/users/User"
+import { StackScreenProps, createStackNavigator } from "@react-navigation/stack"
+import { SettingsScreen } from "@screens/SettingsScreen/SettingsScreen"
+import { ChangePasswordScreen } from "@screens/changePassword/ChangePasswordScreen"
+import { ActivitiesStackParamList } from "@stacks/ActivitiesStack"
+import { useAtomValue } from "jotai"
 import { useHydrateAtoms } from "jotai/utils"
+import React from "react"
 import { View } from "react-native"
-import { TouchableIonicon } from "@components/common/Icons"
+import EditProfileView from "../EditProfileScreen/EditProfileView"
+import ProfileScreenView, { ProfileScreenViewProps } from "../ProfileView"
+import { userAtom } from "../state"
+import { EditProfileDismissButton } from "./EditProfileDismissButton"
+import { HeaderLeftProfile, HeaderRightProfile } from "./ProfileHeaders"
 
 export type ProfileScreensParamsList = {
+  ChangePasswordScreen: undefined
```

Comment:
Create a dedicated settingsStack to navigate to for the settings screens (Settings, Change Password), etc.